src/routes/api.ts

```diff
@@ -88,8 +95,8 @@ function setRoutes(redisClient: redis.RedisClient): express.Router {
             return res.status(500).send(getUserFromTokenResponse.error);
         }
 
-        const playlistName: string = `${sessionData.festivalDisplayName} ${sessionData.festivalYear} - Lineup List`;
-        const playlist: any = await spotifyHelper.getOrCreatePlaylist(access, user.id, playlistName);
+        const playlistName: string = sessionData.playlistName;
```

Comment:
Can you add a null check here? If it's null or undefined default back to the same normal string used above and update the sessionData.playlistName field

Reply:
Done, but I need to do some proper testing of this (really need to sort local HTTPS certs for that) to prove that it works and I haven't broken anything, so feel free to review the code change but I'll leave this PR in draft until tested in a few days.